config/lobby/db/002_update_tables

```diff
@@ -0,0 +1,25 @@
+alter table ta_users alter column email type varchar(254);
+alter table ta_users alter column joined type timestamptz;
+alter table ta_users alter column joined set default current_timestamp;
+alter table ta_users alter column lastlogin type timestamptz;
+alter table ta_users alter column lastlogin set default current_timestamp;
+alter table ta_users alter column admin type boolean using case when admin=0 then false else true end;
+alter table ta_users alter column admin set default false;
```

Comment:
I believe all the `ALTER TABLE` commands applied to a single table can be combined into a single statement by separating the actions with commas.  For example (untested):

```sql
alter table ta_users
  alter column email type varchar(254),
  alter column joined type timestamptz,
  alter column joined set default current_timestamp,
  alter column lastlogin type timestamptz,
  alter column lastlogin set default current_timestamp,
  alter column admin type boolean using case when admin=0 then false else true end,
  alter column admin set default false;
```

See the [Postgres docs](https://www.postgresql.org/docs/9.5/static/sql-altertable.html) for examples.